mqtt_translator/__main__.py

```diff
@@ -6,16 +6,26 @@
 
 def main():
 
-    parser = argparse.ArgumentParser(
+    parser = argparse.ArgumentParser(prog='mqtt-translator',
         description='MQTT topic translator which can act as a MQTT bridge.')
     parser.add_argument('-c', '--config-file', required=True,
                         help='Configuration YAML file')
     parser.add_argument('-v ', '--version',
                         action='version', version='%(prog)s 0.1')
+    parser.add_argument('-d ', '--debug', action="count", default=0,

```

Comment:
Can you set the default to 1?
Info logging will only be at startup. So, we can still see it connects fine.

```suggestion
    parser.add_argument('-d ', '--debug', action="count", default=1,
```